packaging/debian/postinst

```diff
@@ -107,6 +107,36 @@ case "$1" in
     fi
     ;;
 
+    #################
+    # DRLM internal #
+    #################
+
+    # Create internal client to DB
```

Comment:
before making the insert it would be necessary to check if it exists or not? now it is certain that we are starting from a version where in the previous ones it will not be there, but when it is 2.4.13 for example and you come from 2.4.12 it may already exist, but if you come from 2.4.11 you will have to create

Reply:
latest commit controls internal client presence.